src/gui/openfilemanager.cpp

```diff
@@ -64,8 +64,8 @@ static QString findDefaultFileManager()
     QStringList subdirs;
     subdirs << "/applications/"
             << "/applications/kde4/";
-    foreach (QString dir, dirs) {
-        foreach (QString subdir, subdirs) {
+    for (const QString &dir : dirs) {
+        for (const QString &subdir : subdirs) {
```

Comment:
I would constidy dir and subdirs (just instantiate subdirs with the values in the constructor, if you can as a constexpr)